cfe_internal/enterprise/federation/federation.cf

```diff
@@ -278,7 +284,8 @@ bundle agent federation_manage_files
     am_feeder::
       "$(cfengine_enterprise_federation:config.bin_dir)/dump.sh"
         copy_from => default:local_dcp( "$(this.promise_dirname)/../../../templates/federated_reporting/dump.sh" ),
-        perms => default:mog( "700", "root", "root" );
+        perms => default:mog( "700", "root", "root" ),
+        action => if_elapsed("$(cfengine_enterprise_federation:config.dump_interval");
```

Comment:
I don't think that you need to quote the variable , also it looks like you are missing a closing paren `)`  for `if_elapsed`.

```
         action => if_elapsed( $(cfengine_enterprise_federation:config.dump_interval) );
```